db/db_sst_test.cc

```diff
@@ -367,14 +367,16 @@ TEST_F(DBSSTTest, RateLimitedDelete) {
   auto sfm = static_cast<SstFileManagerImpl*>(options.sst_file_manager.get());
   sfm->delete_scheduler()->SetMaxTrashDBRatio(1.1);
 
+  WriteOptions wo;
+  wo.disableWAL = true;
```

Comment:
If I understand correctly, you are disabling WAL here so that the test can pass, as the test is written to the previous specification of WAL files not going though the DeleteScheduler. 
Instead of disabling WAL, why not fix the asserts in the test, to account for the WAL files as well? It might offer more robust test coverage.

Reply:
Its a little complicated as this test makes assumptions about the order of file deletion, which is broken if WAL is enabled. I added a new test specifically to test for WAL deletion by DeleteScheduler to ensure coverage